recipes/libtool/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,163 @@
+from conans import AutoToolsBuildEnvironment, CMake, ConanFile, tools
+from contextlib import contextmanager
+import glob
+import os
+import shutil
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    test_type = "explicit"
+    short_paths = True
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def requirements(self):
+        self.requires(self.tested_reference_str) # Needed as a requirement for CMake to see libraries
+
+    def build_requirements(self):
+        if hasattr(self, "settings_build") and not tools.cross_building(self):
+            self.build_requires(self.tested_reference_str)
+        self.build_requires("automake/1.16.5") # Needed for aclocal called by autoreconf
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
+
+    @contextmanager
+    def _build_context(self):
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self.settings):
+                with tools.environment_append({
+                    "CC": "cl -nologo",
+                    "CXX": "cl -nologo",
+                }):
+                    yield
+        else:
+            yield
+
+    @property
+    def _package_folder(self):
+        return os.path.join(self.build_folder, "package")
+
+    def _build_autotools(self):
+        """ Test autotools integration """
+        # Copy autotools directory to build folder
+        shutil.copytree(os.path.join(self.source_folder, "autotools"), os.path.join(self.build_folder, "autotools"))
+        with tools.chdir("autotools"):
+            # Work around the fact that "used_special_vars" in conans/client/tools/win.py doesn't handle ACLOCAL_PATH
+            aclocal_path = "$ACLOCAL_PATH:" + self.deps_env_info.vars["ACLOCAL_PATH"][0].lower()
+            self.run("ACLOCAL_PATH={} autoreconf --install --verbose -Wall".format(aclocal_path), win_bash=tools.os_info.is_windows)
```

Comment:
```suggestion
            self.run("autoreconf --install --verbose -Wall", win_bash=tools.os_info.is_windows)
```
This simplification may be possible now that unix_path_package_info_legacy() is being used